RELEASE_NOTES.md

```diff
@@ -1,6 +1,14 @@
 # Hipmer release notes
 =========================================
 
+2.4.0
+-----
+* added an authorized user list which can exceed the input size limit of 500 Gbp
+
+2.3.0
```

Comment:
I think we can keep the version at 2.3.0, since it was never promoted to beta or released

Reply:
@mikacashman I think we should add notes about https://github.com/kbaseapps/kbase_hipmer/pull/17/files

Reply:
That is fine with me, that makes this in-line with the ui listed versions as well.  @scanon you good with the release notes from your PR to update to Perlmutter?

Reply:
@mikacashman your updates seem good enough to me! Can we please get this merge in and registered on ci and appdev and then tested on appdev when you get a chance? Thanks